test/unit/app/services/asset_manager/attachment_updater_test.rb

```diff
@@ -4,281 +4,234 @@ class AssetManager::AttachmentUpdaterTest < ActiveSupport::TestCase
   extend Minitest::Spec::DSL
 
   describe AssetManager::AttachmentUpdater do
-    let(:updater) { AssetManager::AttachmentUpdater }
-    let(:update_service) { mock("asset-manager-update-asset-worker") }
-    let(:attachment_data) { attachment.attachment_data }
-    let(:simple_pdf) { File.open(fixture_path.join("simple.pdf")) }
-    let(:attachment) { FactoryBot.create(:file_attachment, file: simple_pdf) }
-    let(:original) { Asset.variants[:original] }
-    let(:thumbnail) { Asset.variants[:thumbnail] }
-    let(:original_asset) { Asset.new(asset_manager_id: "original_asset_manager_id", variant: original, filename: "simple.pdf") }
-    let(:thumbnail_asset) { Asset.new(asset_manager_id: "thumbnail_asset_manager_id", variant: thumbnail, filename: "thumbnail_simple.pdf.png") }
-
-    around do |test|
-      AssetManager.stub_const(:AssetUpdater, update_service) do
-        test.call
-      end
-    end
-
-    before do
-      attachment_data.assets = [original_asset, thumbnail_asset]
-    end
-
-    context "when the attachment has been deleted" do
-      before do
-        attachment.delete
-      end
+    context "when the attachment does not belong to an attachable" do
+      it "does not update the status of any assets" do
```

Comment:
I am not sure what this is meant to test. I believe all classes that include the `Attachable` concern have attachments and `AttachmentData`. Implicitly, they are the attachable. I doubt there is an attachment that does not have an attachable. On top of that, most attachables are editions. Now, there is a funny outlier, which is `PolicyGroup`.  For some reason the factories for attachments do set an attachable by default but it is `PolicyGroup` of all things. We actually decided to leave it be like that so that people are required to specify the attachable when it is relevant in the test. See [factory here](https://github.com/alphagov/whitehall/blob/c015d3cf5ad7e4a5adf3e1629f45f8d6a5671f10/test/factories/attachments.rb#L5). If it makes more sense to change the default to some edition, go for it. 
In this test, it passes because the link header update does not run as it is only set for editions, I think.

Reply:
Yeah I ended up removing this test in the other PR because you're right, there is no way for an attachment not to have an attachable. I did initially try changing it to "not an attachable" as you suggest, but it turns out according to [this test](https://github.com/alphagov/whitehall/blob/c015d3cf5ad7e4a5adf3e1629f45f8d6a5671f10/test/integration/attachment_draft_status_integration_test.rb#L100) the `AttachmentUpdater` should work for a `PolicyGroup` too. So I concluded that the test for not having an attachable did not make any sense.

Reply:
I have removed this test from this PR